src/software/thunderscope/thunderscope.py

```diff
@@ -458,27 +489,51 @@ def configure_robot_diagnostics_layout(self, dock_area, proto_unix_io):
         log_dock = Dock("Logs")
         log_dock.addWidget(self.diagnostics_widgets["log_widget"])
 
+        if not load_fullsystem:
+            self.diagnostics_widgets["robot_view"] = self.setup_robot_view(
+                proto_unix_io, True
+            )
+            robot_view_dock = Dock("RobotView")
+            robot_view_dock.addWidget(self.diagnostics_widgets["robot_view"])
+            self.toggle_robot_connection_signal = self.diagnostics_widgets[
+                "robot_view"
+            ].toggle_robot_connection_signal
```

Comment:
when have BOTH, shouldn't we see the RobotView checkboxes? I don't see it when I run `--run_diagnostics` and `--run_blue`

If your second PR changes how connection to diagnostics/full system works, then I'm okay with letting this through